JiraReporter/JwtAuthenticator.cs

```diff
@@ -0,0 +1,74 @@
+﻿using JWT;
+using RestSharp;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace JiraReporter
+{
+    public class JwtAuthenticator : IAuthenticator
+    {
+        private readonly string jwtToken;
+
+        public JwtAuthenticator(RestRequest request, string sharedSecret)
+        {
+            var path = request.Resource.Split('?')[0];
```

Comment:
@vladichimescu 
Nu inteleg de ce primesti in constructor un RestRequest.

Vei primi un `IRestRequest` cand e momentul sa il autentifici, in metoda Authenticate. In constructor ar trebui sa iti trimiti doar informatiile necesare pentru a face autentificarea.